cfme/middleware/provider.py

```diff
@@ -87,3 +89,14 @@ def num_server(self):
     @num_server.variant('ui')
     def num_server_ui(self):
         return int(self.get_detail("Relationships", "Middleware Servers"))
+
+    def nav_to_provider_view(self):
+        self.load_details(self)
```

Comment:
We automatically suppose that the object's load_details opens the details page of the object, no need to alias it with some specific name.